xcvm/lib/core/src/lib.rs

```diff
@@ -0,0 +1,223 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+extern crate alloc;
+
+mod abstraction;
+mod asset;
+mod instruction;
+mod network;
+mod program;
+mod protocol;
+
+use core::marker::PhantomData;
+
+pub use crate::{asset::*, instruction::*, network::*, program::*, protocol::*};
+use alloc::{collections::VecDeque, vec::Vec};
+
+#[inline]
+pub fn serialize_json<T: serde::Serialize>(
```

Comment:
Do we need to have this? Users can use `serde_json` or `serde-json-wasm` on their choice.

Reply:
Removed